_blog/2023-12-29-using-loco-rust-rails.mdx

```diff
@@ -0,0 +1,316 @@
+In this article we're going to talk about how you can get started with Loco - a new web framework that builds on Axum. We will cover getting started using controllers, migrations, middleware, static files and handling a database.
+
+## Getting Started
+
+To get started, you will need to make sure to have Loco's CLI installed by using the following:
+
+```bash
+cargo install loco-cli
+```
+
+Loco also uses `sea-orm-cli` to carry out database migrations. You can install it using the following:
+
+```bash
+cargo install sea-orm-cli
+```
+
+Now we've installed all of the required packages, we can get initialise our project. We can get started by using `loco new` and then selecting the SaaS application which will give us an app with full functionality. The given name we will be using for the app will be `example-app`. Don't forget to `cd` into the project folder!
```

Comment:
it is probably better to use `example_app` (underscore), as this will be translated to a crate name in Rust, but if everything worked for you and it compiled OK -- never mind